src/popup/components/Modals/ResetWalletModal.vue

```diff
@@ -77,6 +77,9 @@ export default defineComponent({
       disconnectDapps();
 
       await WalletStorage.clear();
+      // TODO: Rethink this approach
+      // It is removing the remaining vuex state
+      await browser.storage.local.clear();
```

Comment:
Maybe we could add this in next release when we will be 100% ready that the migrations works fine?

Reply:
The user won't be able to reset the wallet without it.

Reply:
You're right. So let's leave it.